word2gauss/embeddings.pyx

```diff
@@ -523,6 +523,36 @@ cdef class GaussianEmbedding:
         )
         self.acc_grad_sigma_ptr = &_acc_grad_sigma[0]
 
+
+    def get_phrases_vector(self, phrases, vocab=None):
+        if vocab is None:
+            raise Exception("Cannot convert phrase to a vector without a vocabulary")
+        vec = np.zeros(self.K)
+        if phrases == [''] or phrases == []:
+            return vec
+        else:
+            for ph in phrases:
+                ph_tok = vocab.tokenize(ph)
+                phrase_vec = np.zeros(self.K)
+                for p in ph_tok:
+                    if isinstance(p, basestring):
+                        phrase_vec += self.mu[vocab.word2id(p), :]
+                phrase_vec /= len(ph_tok)
+                vec += phrase_vec
+            vec /= len(phrases)
+            return vec
+
+    def phrases_to_vector(self, target, vocab=None):
+        ''' Input is a list of lists, where target[0] is the list of positive
+        phrases and target[1] is the optional list of negative phrases
+        '''
+        if vocab is None:
```

Comment:
Why not just make this a required argument?  Makes it clear from the function signature that vocab is required.